forum/projects/models.py

```diff
@@ -80,4 +81,62 @@ def clean(self):
+
+        Attributes:
+            funding_id (UUIDField): The unique identifier for the subscription. Automatically generated.
+            project_id (ForeignKey): A reference to the associated project. When the project is deleted, this
+                field will be set to null.
+            investor_id (ForeignKey): A reference to the investor making the subscription. When the investor
+                is deleted, this field will be set to null.
+            funded_amount (DecimalField): The amount of money that has been funded by the investor for the project.
+            funded_at (DateTimeField): The timestamp when the funding was made. Automatically set when the
+                subscription is created.
+            investment_share (DecimalField): The percentage share of the investment in relation to the total
+                required amount of the project. Limited to a maximum of 5 digits with 2 decimal places.
+            is_rejected (BooleanField): Indicates whether the subscription has been rejected. Defaults to False.
+            rejection_reason (CharField): The reason for the rejection if the subscription is rejected.
+
+        """
+
+    funding_id=models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    project_id=models.ForeignKey('Project', on_delete=models.SET_NULL, null=False, related_name='subscribed_projects')
+    investor_id=models.ForeignKey(Investor, on_delete=models.SET_NULL, related_name='projects', db_index=True)
+    funded_amount=models.DecimalField(max_digits=15, decimal_places=2)
+    funded_at=models.DateTimeField(auto_now_add=True, db_index=True)
+    #funding_type. Didn't catch the idea of this field.
+    investment_share=models.DecimalField(max_digits=5, decimal_places=2)
+    is_rejected=models.BooleanField(default=False)
+    rejection_reason=models.CharField(max_length=255, db_index=True)
+
+    def save(self, *args, **kwargs):
+
+        if not self.funded_amount:
+            raise ValidationError(_("Funded amount must be provided."))
+
+        if not self.project_id or not self.project_id.required_amount:
+            raise ValidationError(_("Project must have a valid ID and required amount."))
+
+        self.investment_share = (self.funded_amount / self.project_id.required_amount) * 100
+        super().save(*args, **kwargs)
+
+    def clean(self):
+        super().clean()
+        if self.funded_amount < 0:
+            raise ValidationError(_('Funded amount must be a positive number.'))
+
+        total_share = self.project_id.subscribed_projects.aggregate(models.Sum('investment_share'))[
+                          'investment_share__sum'] or 0
+        if total_share + self.investment_share > 100:
+            raise ValidationError(_('Total investment share for this project cannot exceed 100%.'))
+
+    def __str__(self):
+        return f'Subscription {self.funding_id} by Investor {self.investor_id} for {self.investment_share}% ' \
+               f'of the project'
+
```

Comment:
If you want to ensure that an investor cannot subscribe to the same project multiple times, you can add a unique constraint on the `project_id` and `investor_id` combination
```
class Meta:
    unique_together = ('project_id', 'investor_id')
```